src/templates/user/ico_claim_form.html.tt

```diff
@@ -0,0 +1,50 @@
+    <li>[% l('the Subscriber has read and understood the Information Memorandum of the Company dated 15 November 2017 and in particular has read and understood the Risk Factors detailed therein;') %]</li>
+    <li>[% l('the Subscriber has observed the laws of all relevant jurisdictions, obtained any requisite governmental exchange controls or other consents, complied with all relevant formalities and paid any issue, transfer or other taxes due in connection with its participation in the Auction in any applicable territory and that it has not taken any action which will or may result in the Issuer being in breach of the legal or regulatory requirements of any jurisdiction;') %]</li>
+    <li>[% l('the Subscriber holds an account on the Binary.com website, with an account identifier that shall be identified as the initial holder of the Tokens, and that Subscriber has satisfactorily completed KYC (Know Your Client), AML (Anti-Money Laundering) procedures, and has accurately updated his or her name, address, country of residence, and ID details in such account;') %]</li>
+    <li>[% l('the Subscriber understands that the Tokens are a risky investment, that he/she is investing entirely at his/her own risk, and may lose the entirety of his/her investment; and') %]</li>
+    <li>[% l('the Tokens will be issued on, and the Subscriber will be bound by, the Instrument and the Conditions, including the representations made herein') %]</li>
+</ol>
+<p>[% l('Electronically signed by the Subscriber/for and on behalf of the Subscriber') %]</p>
+<p>
+    <input type="text" disabled="disabled" class="center-text claimer_name" id="name" placeholder="[% l('Name') %]">
+</p>
+<p>[% l('Subscriber') %]</p>
+<p>[% l('Date: ') %] <input type="text" disabled="disabled" class="center-text" id="date_today" placeholder="[% l('Date') %]"></p>
```

Comment:
Please do not rely on trailing space in translations.
How about `<p>[% l('Date') %]: <input ...` to also avoid multiple translations for one word?